dm/master/http_handler.go

```diff
@@ -45,16 +46,27 @@ func getStatusHandle() http.Handler {
 }
 
 // getHTTPAPIHandler returns a HTTP handler to handle DM-master APIs.
-func getHTTPAPIHandler(ctx context.Context, addr string) (http.Handler, error) {
+func getHTTPAPIHandler(ctx context.Context, addr string, securityOpt grpc.DialOption) (http.Handler, error) {
 	// dial the real API server in non-blocking mode, it may not started yet.
-	opts := []grpc.DialOption{grpc.WithInsecure()}
+	opts := []grpc.DialOption{securityOpt}
 	// NOTE: should we need to replace `host` in `addr` to `127.0.0.1`?
 	conn, err := grpc.DialContext(ctx, addr, opts...)
 	if err != nil {
 		return nil, terror.ErrMasterHandleHTTPApis.Delegate(err)
 	}
 
-	gwmux := runtime.NewServeMux()
+	jsonpb := &gateway.JSONPb{
+		EmitDefaults: true,
+		Indent:       "  ",
+		OrigName:     true,
+	}
```

Comment:
this will change the output of HTTP API, the old will output all the result in one line, after this pr will output the same with dmctl. so some integration test need to update

Reply:
Is this what you want to do or you must to do?

Reply:
It is not necessary, I just found these code in another project,  and this will make the output of HTTP response more clear, so add here